### PR TITLE
Minor grammatical fixes

### PR DESCRIPTION
**Pull request by sebgl**

### CODE REVIEW

File: engine/api/README.md [hunk 1]
```diff
@@ -3,23 +3,23 @@
...
 ## CDS API Third-parties
 
-At the minimum, CDS needs a PostgreSQL Database >= 9.4. But for serious usage your may need :
+At the minimum, CDS needs a PostgreSQL Database >= 9.4. But for serious usage your may need:
 
-- A [Redis](https://redis.io) server or sentinels based cluster used as a cache and session store
-- A LDAP Server for authentication
-- A SMTP Server for mails
+- A [Redis](https://redis.io) server or sentinels based cluster used as cache and session store
+- An LDAP Server for authentication
```
Comment (fsamin) [inline engine/api/README.md]: nope

File: engine/api/README.md [hunk 2]
```diff
@@ -3,23 +3,23 @@
...
+- An SMTP Server for mails
```
Comment (fsamin) [inline engine/api/README.md]: nope

File: engine/api/README.md (hunk 1 above)
Comment (sebgl) [inline engine/api/README.md]: http://www.grammar.com/a-vs-an-when-to-use/

https://en.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol  <- ctrl+f  "an LDAP" vs. "a LDAP"  ;)

Comment (fsamin) [inline engine/api/README.md]: you got me !

**Pull request merged**